src/NHSD.BuyingCatalogue.Ordering.Domain/OrderStatus.cs

```diff
@@ -1,9 +1,8 @@
 ﻿namespace NHSD.BuyingCatalogue.Ordering.Domain
 {
-    public sealed class OrderStatus
+    public enum OrderStatus : int
```

Comment:
I dont think you have to do :int, as it does it as defualt

Reply:
👍